### PR TITLE
[ACM-8527]: Adding known issue for hosted clusters in disconnected environment

### PR DESCRIPTION
**Pull request by lahinson**
Related Jira: https://issues.redhat.com/browse/ACM-8527

### CODE REVIEW

**Comment by lahinson** [comment]
@jparrill and @zanetworker PTAL when you can. Note that I pointed to the DNS docs for dual stack that will be published in the official docs next week instead of the netlify docs. Thanks!

**Review by zanetworker — APPROVED** [review]

File: clusters/release_notes/known_issues.adoc [hunk 1]
```diff
@@ -484,4 +484,15 @@ When you use the console to add a new node pool to an existing hosted cluster, t
 
 You can specify a custom ingress domain by using the `ClusterDeployment` resource while installing a managed cluster, but the change is only applied after the installation by using the `SyncSet` resource. As a result, the `spec` field in the `clusterdeployment.yaml` file displays the custom ingress domain you specified, but the `status` still displays the default domain.
 
+[#hosted-cluster-dual-stack]
+=== DNS issues in hosted clusters configured for a dual-stack, disconnected environment
```
Comment (jparrill) [inline clusters/release_notes/known_issues.adoc]: Could we add something like "Potential DNS Issues" or similar? Sounds alarming

Comment (lahinson) [reply clusters/release_notes/known_issues.adoc]: Sure, I'll add "Potential" ✔️

Comment (lahinson) [reply clusters/release_notes/known_issues.adoc]: Done

**Review by jc-berger — APPROVED** [review]
/lgtm

**Pull request merged**